eslint/eslint-patch/src/usage.ts

```diff
@@ -5,3 +5,5 @@ throw new Error(
   'The @rushstack/eslint-patch package does not have a default entry point.' +
     ' See README.md for usage instructions.'
 );
+
+export = {};
```

Comment:
```suggestion
export {};
```
TypeScript will emit code to identify it is as the module type declared in tsconfig.json, which in this case is CommonJS.

Reply:
+1  

`export =` is deprecated.

Reply:
Thanks! I've checked this continues to work as expected with the vite build, and can confirm the warning doesn't appear.